packages/react/src/components/cookieConsent/consentGroup/ConsentGroup.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import { VisuallyHidden } from '@react-aria/visually-hidden';
+
+import {
+  ConsentGroup as ConsentGroupType,
+  CookieConsentContext,
+  useCookieConsentActions,
+} from '../CookieConsentContext';
+import styles from '../CookieConsent.module.scss';
+import { Checkbox } from '../../checkbox/Checkbox';
+import { useAccordion } from '../../accordion';
+import { IconAngleDown, IconAngleUp } from '../../../icons';
+import { Card } from '../../card/Card';
+import ConsentGroupDataTable from '../consentGroupDataTable/ConsentGroupDataTable';
+
+function ConsentGroup(props: { group: ConsentGroupType; isRequired: boolean; id: string }): React.ReactElement {
+  const { group, isRequired, id } = props;
+  const { isOpen, buttonProps, contentProps } = useAccordion({
+    initiallyOpen: false,
+  });
+  const groupConsents = group.consents;
+  const cookieConsentContext = useContext(CookieConsentContext);
+  const triggerAction = useCookieConsentActions();
+  const areAllApproved = isRequired || cookieConsentContext.areGroupConsentsApproved(groupConsents);
```

Comment:
`cookieConsentContext.areGroupConsentsApproved(groupConsents);`

Does this mean that the consents are approved already, or that user is about to approve them (checkbox is checked)?